pkg/console/status/status.go

```diff
@@ -64,6 +64,36 @@ func HandleUpgradable(typePrefix string, reason string, err error) v1helpers.Upd
 	return v1helpers.UpdateConditionFn(condition)
 }
 
+func (c *StatusHandler) ResetConditions(conditions []operatorsv1.OperatorCondition) []v1helpers.UpdateStatusFunc {
+	updateStatusFuncs := []v1helpers.UpdateStatusFunc{}
+	for _, condition := range conditions {
+		klog.V(2).Info("\nresetting condition: ", condition.Type)
```

Comment:
Should we move this output to each "if" block so that we can also output what we are resetting the value to?

```suggestion
```